src/tests/ftest/util/general_utils.py

```diff
@@ -1332,7 +1346,7 @@ def report_errors(test, errors):
             len(errors), "" if len(errors) == 0 else "s"))
         test.fail(error_msg)
 
-    test.log.info("No errors detected.")
+    test.log.info("Test passed")
```

Comment:
This function is used by other tests as well, and generally might be called multiple times. So "Test passed" isn't always correct. I.e. if the test continues to do more work and then calls report_errors again

Reply:
Fixed with commit 6e503c582